package.json

```diff
@@ -27,14 +27,17 @@
   ],
   "scripts": {
     "clean": "rm -rf ./lib ./dist",
-    "build": "tsc && rollup -c",
+    "build": "npm run build:main && npm run build:test && npm run build:examples",
+    "build:main": "tsc && rollup -c ./build/main.config.js",
+    "build:test": "rollup -c ./build/test.config.js",
+    "build:examples": "rollup -c ./build/examples.config.js",
```

Comment:
It seems like both `test` and `examples` depend on `main`. Will the `watch` script always perform the appropriate build as files in test or examples change?

Reply:
`examples` should be standalone AFAIK (just builds some iron-components), but you're right this isn't quite what is expected for test (changing a test file in src/* would trigger a build of only the main component) -- I'll give this component another think